### PR TITLE
SDK - Components - Made it easier to access component spec classes

### PR DESCRIPTION
**Pull request by Ark-kun**
Make the classes like `ComponentSpec` accessible.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2860)

### CODE REVIEW

File: sdk/python/kfp/components/structures/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .._structures import *
```
Comment (numerology) [inline sdk/python/kfp/components/structures/__init__.py]: Can we rename the current structures dir to schema, then we just rename _structures.py to structures.py. I think it'll be better to let the package path aligned with the dir layout.

Comment (Ark-kun) [reply sdk/python/kfp/components/structures/__init__.py]: We might do this in future, but for now I'm a bit cautious to make the `kfp.components._structures` unavailable. If it's OK, I'd propose changing that in a later release.

Comment (numerology) [reply sdk/python/kfp/components/structures/__init__.py]: Sure.

**Pull request merged**